src/components/Home.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function Home() {
+    return (
+        <>
```

Comment:
You can omit the fragment(<></>) and just put:
`<h1>Welcome to GhibliApp</h1>`
You use fragments to wrap multiple elements. In this case there is only one element.